notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<p>Is the "10M" claim still true after the recent changes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>Yes, it's approximately 11M rows in NYC Green Taxi dataset in 2017.</p>